packages/sanes-chrome-extension/src/routes/signup/index.e2e.spec.ts

```diff
@@ -0,0 +1,48 @@
+import puppeteer, { Browser, Page } from 'puppeteer';
+import { EXTENSION_HEIGHT, EXTENSION_WIDTH } from '../../theme/constants';
+import { travelToSignupNewAccountStep } from './test/travelToSignup';
+import { submitAccountFormE2E, handlePassPhrase2E, handleSecurityHintE2E } from './test/fillSignupForm';
+import { randomString } from '../../utils/test/random';
+import { withChainsDescribe } from '../../utils/test/testExecutor';
+
+withChainsDescribe(
+  'DOM > Signup route',
+  (): void => {
+    let browser: Browser;
+
+    beforeEach(async (): Promise<void> => {
+      const CRX_PATH = require('path').join(__dirname, '../../../build');
```

Comment:
This is the second e2e test done, so this is the time to refactor it. In the same way, we have a general `createDom` method, we should create a `createBrowser` method and reuse it in both tests.